recipes/irida-uploader/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "irida-uploader" %}
+{% set version = "0.1" %}
+
+package:
+  name: '{{ name }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/phac-nml/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 8080b411c0fcbc40d7792c65f1c9736186b0f5f436dc7115e0979cb80de1ea3e
+  patches: 
+    - upload_run.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python =3.6
+    - pip
+    - make
+    - setuptools
+    - rauth==0.7.3
+    - autopep8==1.4
+    - pycodestyle 
+    - requests==2.21.0
+    - appdirs==1.4.3
+    - pytest
+    - mkdocs==1.0.0
+    - cerberus
+    - argparse
```

Comment:
Argparse has been built in since python 2.7. pytest is probably only needed for testing (likewise with things like pycodestyle and autopep8).